custom/conf/app.example.ini

```diff
@@ -207,6 +207,8 @@ DEFAULT_SHOW_FULL_NAME = false
 SEARCH_REPO_DESCRIPTION = true
 ; Whether to enable a Service Worker to cache frontend assets
 USE_SERVICE_WORKER = true
+; show `system administrator` tag on comment header
+SHOW_SYS_ADMIN_COMMENT_TAG = false
```

Comment:
Is this setting really needed?

Reply:
Because  I think the ``system administrator`` tag is not so necessary, Only when the system administrator want user know thire Identity when read the comment, this tag is usefull. In most cases, this tag is not usefull. so I set it as an optional choice.

Reply:
Maybe than don't add it at all, I don't see much use case for that

Reply:
@lafriks done